app/src/main/java/org/xwiki/android/sync/activities/GroupListAdapter.java

```diff
@@ -35,29 +36,64 @@
 import java.util.ArrayList;
 import java.util.List;
 
+/**
+ * {@link android.widget.Adapter} which can be used to show groups
```

Comment:
Like with the file headers it would be better to follow XWiki codestyle regarding class javadoc, see http://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HUseversionandsincejavadoctags.